Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset

```diff
@@ -16,7 +16,7 @@ MonoBehaviour:
   HasSourceProject: 0
   HasPlatforms: 0
   sourceProjectPath: 
-  sourceBankPath: FMOD\Desktop\Desktop
```

Comment:
Just a question, this is just a Windows and Mac thing right? I find myself having to change the source everytime I merge into my branch because I don't push this file. If this means that people on Windows now have to do it then mwahahahahahaha.

Reply:
ME TOO! Whenever I pull.. I need to redo my settings so I'm not sure!

Reply:
no idea about this... @masonachu any ideas?